src/errors.rs

```diff
@@ -36,10 +36,10 @@ impl From<PointError> for DkgError {
 /// Errors which can happen during signature aggregation
 pub enum AggregatorError {
     #[error("bad poly commitment length (expected {0} got {1})")]
-    /// The polynomial commitment was the wrong size
+    /// The number of polynomial commitments was wrong (no longer used)
```

Comment:
Ohh, this is not removed because it's breaking change right?

Reply:
Exactly